### DISCUSSION TITLE
Panel Upload  - not possible anymore?

### DISCUSSION THREAD

**Discussion by fbit-itc** [RomanKubiak/ctrlr#428] [General]
Hello everybody,

a week ago I wanted to publish new panel releases on ctrlr.org, but login is not possible and I did not get an email response from Roman.
From what I could read here:
1. logins into ctrlr.org have been closed (bc of spam) so even the developpers cannot upload panels or modify panel pages?
2. there is actually no working solution for publishing new or updated panels

Is this correct?
Is there any attempt to change the latter in the nearby future, e.g. create a new web interface for publishing panels?
Because if not, I would create my own website for my panels. In this situation it would be cool if a least we could put a hint or link on the panel pages pointing to an other storage with a new panel - in the actual situation people will come to ctrlr.org, and, not knowing of the new panel, download the outdated version ...

cheers
F

**Comment by damiensellier** [comment]
Roman does not answer to his email. Ctrlr.org is bloated with spam and Roman removed the write access to anyone more than a year ago. Everybody is publishing or hosting their panels on their own webpages or repository (github or sourceforge). I really liked the idea of a place like ctrlr.org to gather all the panels but it seems to require to much maintenance for Roman. 
You can post a discussion about your panel here with screencaps and zip files anyway.

**Reply by fbit-itc** [reply]
thank you so much. So let's hope Roman is fine. Who knows, maybe he got married, 'babied' and has to get up 10 times each night ...

Ok, it is like it is and we have to find ways to communicate that panels on ctrlr.org are outdated...

**Comment by Godlike-Productions** [comment]
I think Roman created this repo for this, but there are none uploaded yet.

https://github.com/RomanKubiak/Panels/tree/1aaeb12ad9cd96a16cddb37fd2a2c8ca98f87a01
